pandas/core/panel.py

```diff
@@ -679,6 +679,8 @@ def _combine(self, other, func, axis=0):
             return self._combine_frame(other, func, axis=axis)
         elif np.isscalar(other):
             return self._combine_const(other, func)
+        else:
+            raise NotImplementedError('This method has not been implemented yet. ')
```

Comment:
@jreback How about `[op-method-name] has not been implemented in Panel class yet. Please wait for the next releast`?